tqdm/_tqdm.py

```diff
@@ -24,24 +34,28 @@ def format_meter(n, total, elapsed):
         total = None
 
     elapsed_str = format_interval(elapsed)
-    rate = '%5.2f' % (n / elapsed) if elapsed else '?'
+    rate = '{0:5.2f}'.format(n / elapsed) if elapsed else '?'
 
     if total:
         frac = float(n) / total
 
-        N_BARS = 10
-        bar_length = int(frac*N_BARS)
-        bar = '#'*bar_length + '-'*(N_BARS-bar_length)
-
-        percentage = '%3d%%' % (frac * 100)
+        N_BARS = 32
```

Comment:
Why `N_BARS = 32`, it would be nice to have it at `10` by default and make it a parameter, don't you think ?